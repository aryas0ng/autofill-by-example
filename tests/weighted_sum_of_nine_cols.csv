1,0,9,0,2,1,7,2,3,114
2,4,8,5,1,16,3,6,3,229
2,3,7,4,0,3,6,1,34,181
3,8,6,3,2,7,5,8,4,224
10,10,5,2,1,8,11,6,4,303
5,5,4,1,0,7,2,1,6,161
4,4,3,0,9,1,1,21,4,187
19,4,2,2,8,8,5,3,1,323
2,21,1,1,7,4,4,4,14,284
1,21,0,0,6,1,4,6,5,240
9,10,4,6,5,9,6,15,1,
7,7,3,5,4,8,1,3,1,
4,4,2,4,3,2,6,4,1,
3,3,1,3,2,6,4,1,1,
12,3,0,2,1,13,1,1,11,
4,16,1,1,0,1,4,9,12,
1,16,0,0,2,7,5,4,16,
26,27,5,3,1,10,13,2,16,
8,8,4,2,0,2,5,2,17,
3,3,3,1,3,3,1,3,2,
3,3,2,0,2,1,11,7,2,
8,11,1,4,1,1,15,23,2,
4,4,0,3,0,1,2,3,20,
6,5,9,2,5,1,2,8,20,
5,4,8,1,4,1,2,3,27,
4,3,7,0,2,17,2,6,4,
3,2,6,6,1,2,4,7,4,
2,1,5,5,0,21,43,3,5,
1,0,4,4,2,26,5,1,54,
0,2,3,3,1,4,6,8,6,
3,1,2,2,3,7,9,6,6,
2,0,1,1,3,8,3,4,7,
1,1,0,0,4,4,4,4,8,
0,0,4,3,46,7,2,17,8,
6,1,3,2,5,9,5,5,8,
5,0,2,1,6,5,5,11,15,