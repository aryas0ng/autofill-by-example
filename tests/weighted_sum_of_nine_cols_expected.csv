1,0,9,0,2,1,7,2,3,114
2,4,8,5,1,16,3,6,3,229
2,3,7,4,0,3,6,1,34,181
3,8,6,3,2,7,5,8,4,224
10,10,5,2,1,8,11,6,4,303
5,5,4,1,0,7,2,1,6,161
4,4,3,0,9,1,1,21,4,187
19,4,2,2,8,8,5,3,1,323
2,21,1,1,7,4,4,4,14,284
1,21,0,0,6,1,4,6,5,240
9,10,4,6,5,9,6,15,1,335
7,7,3,5,4,8,1,3,1,232
4,4,2,4,3,2,6,4,1,156
3,3,1,3,2,6,4,1,1,125
12,3,0,2,1,13,1,1,11,217
4,16,1,1,0,1,4,9,12,223
1,16,0,0,2,7,5,4,16,214
26,27,5,3,1,10,13,2,16,607
8,8,4,2,0,2,5,2,17,220
3,3,3,1,3,3,1,3,2,116
3,3,2,0,2,1,11,7,2,128
8,11,1,4,1,1,15,23,2,293
4,4,0,3,0,1,2,3,20,122
6,5,9,2,5,1,2,8,20,240
5,4,8,1,4,1,2,3,27,202
4,3,7,0,2,17,2,6,4,209
3,2,6,6,1,2,4,7,4,164
2,1,5,5,0,21,43,3,5,315
1,0,4,4,2,26,5,1,54,246
0,2,3,3,1,4,6,8,6,116
3,1,2,2,3,7,9,6,6,149
2,0,1,1,3,8,3,4,7,102
1,1,0,0,4,4,4,4,8,81
0,0,4,3,46,7,2,17,8,352
6,1,3,2,5,9,5,5,8,189
5,0,2,1,6,5,5,11,15,167